ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudDestination.java

```diff
@@ -252,6 +252,24 @@ private boolean uploadIfNotExists(BlobId blobId, long inputLength, CloudBlobMeta
     }
   }
 
+  @Override
+  public void downloadBlob(BlobId blobId, OutputStream outputStream) throws CloudStorageException {
```

Comment:
Please add test cases in both AzureCloudDestinationTest and AzureIntegrationTest.  The latter in particular can serve as a real-world correctness test for the complete upload/download cycle.

Reply:
sure.